WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -625,6 +627,7 @@ static GSourceFuncs _handlerIntervention =
             Core::JSON::Boolean AllowFileURLsCrossAccess;
             Core::JSON::Boolean SpatialNavigation;
             Core::JSON::EnumType<HTTPCookieAcceptPolicyType> CookieAcceptPolicy;
+            Core::JSON::ArrayType<JsonObject> EnvVars;
```

Comment:
Let's please use a strong type here and not JsonObject. That is easier to understand and type safe (and more efficient)